Print_A_Bot/controls/models.py

```diff
@@ -11,13 +13,18 @@ class LightShow(models.Model):
     def __str__(self):
         return self.name
 
+
 class LightShowStep(models.Model):
     show = models.ForeignKey(LightShow)
     light = models.IntegerField(choices=settings.LED_CHOICES)
-    red = models.IntegerField()
-    green = models.IntegerField()
-    blue = models.IntegerField()
-    order = models.IntegerField()
+    color = RGBColorField(default="green")
+    red = models.IntegerField(blank=True, null=True)
```

Comment:
the individual colors may be able to be removed entirely with `color` field

either way they all should have `help_text` describing their intended useage